src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: $main-color;
```

Comment:
Use sass nesting

Reply:
Fixed.
But I used nesting only for pseudo-classes and pseudo-elements on purpose. Because then it's easier to find proper selectors using ctrl+f in your styles file. I thought it's a good practice.